e2e/tests/components/dashboard/dashboard.e2e-spec.ts

```diff
@@ -336,6 +336,25 @@ describe('Dashboard Tests', () => {
         await page.checkAnnouncerText('Moving and resizing cancelled. Dashboard with 4 columns, containing 4 panels. Usage Analytics panel in row 0, column 0, is 4 columns wide and 2 rows high. Service panel in row 2, column 0, is 2 columns wide and 1 rows high. Users panel in row 2, column 2, is 1 columns wide and 1 rows high. Alert panel in row 2, column 3, is 1 columns wide and 1 rows high. Press space to move and resize the Usage Analytics panel.');
     });
 
+    it('should allow widgets to be moved while in stacked mode', async () => {
+        // resize page so in stacked mode
+        await browser.driver.manage().window().setSize(400, 600);

```

Comment:
Since there are now multiple stacked mode tests, I would suggest creating a child `describe` for stacked mode and call `setSize` in its `beforeEach` hook so that it does not need to be repeated for each test.